.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+exclude = nikola/data/themes/base/messages
```

Comment:
Could you put this in setup.cfg instead? One fewer file to remember we put configuration in :-)

Reply:
Thanks for the advice - I just moved it to `setup.cfg`